### PR TITLE
Create ParseParameters functions for BucketClass and BucketAccessClass

### PR DESCRIPTION
**Pull request by aarongalang**
created a new file in azureutils and added BucketClass/BucketAccessClass parameter parsing. Also added constants, and enums.

### CODE REVIEW

File: pkg/constant/general.go [hunk 1]
```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package azureutils
+package constant
```
Comment (mukhoakash) [inline pkg/constant/general.go:14]: This file is not related to any constants. We should keep it in the utils folder itself

File: pkg/azureutils/cosi_api_ops.go [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+type BucketAccessClassParameters struct {
+	bucketUnitType    cons.BucketUnitType
+	region            string
+	signedversion     string
+	signedPermissions cons.SignedPermissions
+	signedExpiry      int
+	signedResouceType cons.SignedResourceType
+}
+
+func ParseBucketClassParameters(parameters map[string]string) (BucketClassParameters, error) {
```
Comment (mukhoakash) [inline pkg/azureutils/cosi_api_ops.go:41]: `func parseBucketClassParameters`. Don't want this function to be publicly available to other pkgs.

File: pkg/azureutils/cosi_api_ops.go [hunk 3]
```diff
@@ -0,0 +1,209 @@
+package azureutils
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	cons "project/azure-cosi-driver/pkg/constant"
```
Comment (mukhoakash) [inline pkg/azureutils/cosi_api_ops.go:8]: Changed `cons` to use the default pkg name `constant`

File: pkg/azureutils/cosi_api_ops.go [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+	BCParams := BucketClassParameters{}
+	for k, v := range parameters {
+		switch strings.ToLower(k) {
+		case cons.BucketUnitTypeField:
+			//determine unit type and set to container as default if blank
+			switch strings.ToLower(v) {
+			case "container":
+				BCParams.bucketUnitType = cons.Container
+			case "":
+				BCParams.bucketUnitType = cons.Container
```
Comment (mukhoakash) [inline pkg/azureutils/cosi_api_ops.go:51]: `case "container", ""`

File: pkg/azureutils/cosi_api_ops.go (hunk 2 above)
Comment (aarongalang) [reply pkg/azureutils/cosi_api_ops.go]: function no longer available to other pkgs.

File: pkg/constant/general.go (hunk 1 above)
Comment (aarongalang) [reply pkg/constant/general.go]: moved the folder back to utils and renamed back to const.go

**Comment by mukhoakash** [comment]
@aarongalang Please ensure that the code is linted correctly before merging.

File: pkg/azureutils/cosi_api_ops.go (hunk 3 above)
Comment (aarongalang) [reply pkg/azureutils/cosi_api_ops.go]: cosi_api_ops.go now uses default pkg name constant

File: pkg/azureutils/cosi_api_ops.go (hunk 4 above)
Comment (aarongalang) [reply pkg/azureutils/cosi_api_ops.go]: case "container" and case "" now combined

**Review by mukhoakash — APPROVED** [review]
